src/Language/DslTransformer.php

```
@@ -0,0 +1,165 @@
+                continue;
+            }
+
+            if (str_starts_with($line, 'schema')) {
+                $version = trim(substr($line, strlen('schema')));
+                if ($version !== '') {
+                    $schemaVersion = SchemaVersion::from($version);
+                }
+                continue;
+            }
+
+            if (preg_match('/^type\s+(\w+)/', $line, $m)) {
```

PHPStan (2.1.15)

Only booleans are allowed in an if condition, int|false given.


(if.condNotBoolean)